integration/framework.go

```diff
@@ -332,7 +332,12 @@ func (f *framework) initGuestClientset() error {
 	return nil
 }
 
-func (f *framework) waitForAPIUp() error {
+func (f *framework) WaitForAPIUp() error {
+	if f.guestCS == nil {
+		if err := f.initGuestClientset(); err != nil {
+			return microerror.Maskf(err, "unexpected error initializing guest clientset")
+		}
+	}
```

Comment:
Shouldn't that `if` be inside `initGuestClientset`? That way you'll avoid repetition if you need to call `initGuestClientset` in another method.

Reply:
Indeed, makes sense.